debian/rules

```diff
@@ -31,6 +31,9 @@ override_dh_auto_install:
 	install -D -m0755 debian/startup/kolibri.init $(CURDIR)/debian/kolibri/etc/init.d/kolibri
 	install -D -m0644 debian/startup/kolibri.default $(CURDIR)/debian/kolibri/etc/default/kolibri
 	install -D -m0644 debian/startup/kolibri.service $(CURDIR)/debian/kolibri/lib/systemd/system/kolibri.service
+	# remove sentry_sdk files depending on Python >= 3.5:
+	rm -f $(CURDIR)/debian/kolibri/usr/lib/python3/dist-packages/kolibri/dist/sentry_sdk/integrations/aiohttp.py
+	rm -f $(CURDIR)/debian/kolibri/usr/lib/python3/dist-packages/kolibri/dist/sentry_sdk/integrations/sanic.py
```

Comment:
Could this be done as a quilt patch to align with other patch sets?

Reply:
Probably it could be done, but we are not modifying any file from the kolibri source code. We are removing a couple of files from an external library. I don't think that's the purpose of using quilt. debian/rules is a very appropiate place to add/remove external files.

Reply:
You choose, was just thinking that we are now doing the same thing in two different ways.

Reply:
See: https://github.com/learningequality/kolibri-installer-debian/tree/master/debian/patches

Reply:
ok, for standarization reasons I have changed it to use quilt to remove these files

Reply:
I have created https://github.com/learningequality/kolibri-installer-debian/pull/68 to remove patches that are not in used currently.